### PR TITLE
Clean up gridworld codes and add comments to reinforce code

### PR DESCRIPTION
**Pull request by Hyeokreal**
Do not merge yet. I need feedback from members.

### CODE REVIEW

File: 1-grid-world/2-value-iteration/value_iteration.py [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
         # 2-d list for the value function
         self.value_table = [[0.0] * env.width for _ in range(env.height)]
+        self.discount_factor = 0.9
 
     # get next value function table from the current value function table
     def value_iteration(self):
-        next_value_table = [[0.0] * \
-                            self.env.width for _ in range(self.env.height)]
+        next_value_table = [[0.0] * self.env.width for _ in
+                            range(self.env.height)]
```
Comment (keon) [inline 1-grid-world/2-value-iteration/value_iteration.py]:
```
next_value_table = [[0.0] * self.env.width 
                                for _ in  range(self.env.height)]
```
how about this bro

Comment (Hyeokreal) [inline 1-grid-world/2-value-iteration/value_iteration.py]: I did what you suggested! Thanks bro

File: 1-grid-world/7-reinforce/reinforce_agent.py [hunk 2]
```diff
@@ -88,37 +99,31 @@ def train_model(self):
...
+        # fresh env
         state = env.reset()
         state = np.reshape(state, [1, 15])
 
         while not done:
-            if agent.render:
-                env.render()
             global_step += 1
-
+            # get action for the current state and go one step in environment
```
Comment (keon) [inline 1-grid-world/7-reinforce/reinforce_agent.py]: I think this exceeds 80 char limit.
Let's split it into two lines.

Comment (Hyeokreal) [inline 1-grid-world/7-reinforce/reinforce_agent.py]: No, it doesn't. 
There are about 3 char spaces left.

**Pull request merged**